### PR TITLE
fix(@desktop/settings): content on the right side for all subsections need to have the same geometry

### LINKED ISSUE #5650: Settings - content on the right side for all subsections need to have the same geometry
All subsection should have the same form.
Headers should not scroll and should always be visible. 
The content is the only part that should scroll.
Width of the content on the left side should be fixed to 560px, margin from the left side 48px, from the app top 56px, from the app bottom edge 56px, on the right side of the content part we will add some hints later (that's why we need it now clean).

### PR DESCRIPTION
**Pull request by saledjenic**
The most important thing added here is `SettingsContentBase` component. It defines high level structure for all subsections in the settings section (all of them should inherit it). Having that whenever we need to change something global for settings section like update margins, title font, color, back button... we will do it from a single place with a single change and affect all subsections. Also `getNameForSubsection` function is added to `ProfileSectionStore` so updating subsection name in a single place will be visible to all corresponding places. Section names are fixed now and only scrollable part is subsection content. 

`EnsView` is not affected by these changes, since it has completely different structure and that should be handled via separate ticket.

Changes done here refers only to a global structure (functionality is not touched at all). We have the same structure everywhere now with minimal changes (even some files look like they are completely changed, that's because of code formatting). Each section should be improved (there are lot of mixing using anchoring, layouts, fixed and relative things), this PR doesn't fix details inside subsections.

Fixes #5650

https://user-images.githubusercontent.com/86303051/167399066-39674c7c-2898-46a1-b31d-facea84789b9.mov

### CODE REVIEW

File: ui/app/AppLayouts/Profile/ProfileLayout.qml [hunk 1]
```diff
@@ -24,10 +25,14 @@ StatusAppTwoPanelLayout {
     }
 
     QtObject {
-        id: _internal
-        readonly property int contentMaxWidth: 624
-        readonly property int contentMinWidth: 450
-        property int profileContentWidth: Math.max(contentMinWidth, Math.min(profileContainer.width * 0.8, contentMaxWidth))
+        id: d
```
Comment (alaibe) [inline ui/app/AppLayouts/Profile/ProfileLayout.qml:28]: nit: what is `d` ? maybe something more meaningful?

File: ui/app/AppLayouts/Profile/ProfileLayout.qml [hunk 2]
```diff
@@ -36,100 +41,198 @@ StatusAppTwoPanelLayout {
...
+        StatusBanner {
+            id: banner
+            anchors.top: parent.top
+            anchors.left: parent.left
+            anchors.right: parent.right
+            visible: {
+                if(profileContainer.currentIndex === Constants.settingsSubsection.wallet &&
+                        profileView.store.walletStore.areTestNetworksEnabled)
+                    return true
+                return false
```
Comment (alaibe) [inline ui/app/AppLayouts/Profile/ProfileLayout.qml:56]: nit: could be simplified to: 
```
visible: profileContainer.currentIndex === Constants.settingsSubsection.wallet && profileView.store.walletStore.areTestNetworksEnabled
```

File: ui/imports/utils/Constants.qml [hunk 3]
```diff
@@ -149,6 +149,41 @@ QtObject {
         ]
     }
 
+    readonly property QtObject settingsSection: QtObject {
+        readonly property int itemSpacing: 10
+        readonly property int mainHeaderFontSize: 28
+
+//        readonly property int radius: 8
```
Comment (alaibe) [inline ui/imports/utils/Constants.qml:156]: why is this commented? can it be removed, or still needed?

**Review by alaibe — APPROVED** [review]
Nice work!

**Comment by alaibe** [comment]
This should also fixes #5568

File: ui/app/AppLayouts/Profile/ProfileLayout.qml (hunk 1 above)
Comment (stefandunca) [reply ui/app/AppLayouts/Profile/ProfileLayout.qml:28]: @alaibe this is a common Qt pattern to hide implementation details. See here for more details: https://wiki.qt.io/D-Pointer#The_d-pointer

Comment (saledjenic) [reply ui/app/AppLayouts/Profile/ProfileLayout.qml:28]: @alaibe in all qml components defined by Qt `d` is used as `id` of `QtObject` containing private properties (I think it stands for "details"), check [here](https://doc.qt.io/archives/qt-4.8/qml-coding-conventions.html#private-properties).

File: ui/app/AppLayouts/Profile/ProfileLayout.qml (hunk 2 above)
Comment (saledjenic) [reply ui/app/AppLayouts/Profile/ProfileLayout.qml]: @alaibe tbh I just copied that code as it is, anyhow I updated it now as you suggested.

File: ui/imports/utils/Constants.qml (hunk 3 above)
Comment (saledjenic) [reply ui/imports/utils/Constants.qml]: @alaibe thanks for this, it's leftover, removed now.

**Review by abhi-status — APPROVED** [review]

**Comment by abhi-status** [comment]
https://ethstatus.testrail.net/index.php?/runs/view/9153&group_by=cases:section_id&group_order=asc

**Pull request merged**